### PR TITLE
Add happy path spec for number input widget

### PR DESCRIPTION
**Pull request by ajith-k-v**
Fixes: #4067

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2uw67c7 Add HappyPath Spec for Number Input](https://app.clickup.com/t/2uw67c7)

**Comment by ajith-k-v** [comment]
@emidhun Please review

Screenshot 2022-09-24 at 9 41 12 AM

File: cypress-tests/cypress/e2e/editor/widget/buttonHappyPath.cy.js [hunk 1]
```diff
@@ -75,12 +77,7 @@ describe("Editor- Test Button widget", () => {
     cy.get(commonWidgetSelector.draggableWidget(data.widgetName)).click();
     cy.verifyToastMessage(commonSelectors.toastMessage, data.alertMessage);
 
-    openEditorSidebar(data.widgetName);
-    openAccordion(commonWidgetText.accordionGenaral);
-    addAndVerifyTooltip(
```
Comment (emidhun) [inline cypress-tests/cypress/e2e/editor/widget/buttonHappyPath.cy.js:80]: I think we can keep the code, Sice it fails because of a bug.

**Review by emidhun — CHANGES_REQUESTED** [review]
Can we keep the failing code on the codebase, since the failures are due to application issues.

**Review by emidhun — APPROVED** [review]
LGTM

**Pull request merged**